src/lib/EasyEdit.jsx

```diff
@@ -110,6 +110,7 @@ export default class EasyEdit extends React.Component {
                 onChange={this.onChange}
                 type={type}
                 attributes={attributes}
+                instructions={instructions}
```

Comment:
These `instructions` props are no longer required here if the functionality has been extracted to `EasyEdit.jsx`